hr_employee_relative/models/hr_employee.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    relatives = fields.One2many(
+        string='Relatives',
+        comodel_name='hr.employee.relative',
```

Comment:
Would it make sense to do a many2many to `res.partner` instead? That table already contains all possibly required fields to identify somebody, and addons that add the missing ones you need...

Reply:
You need to indicate also the type of relation, so  you need a new model.

Reply:
It's not a `res.partner` in a matter of speaking, especially when it comes to notes like "what gifts relatives prefer" if such policies exist in company, plus relation type is required

Reply:
@alexey-pelykh any way, it would be great to have a link to res.partner in that one2many table for relatives with persons.

Reply:
OK I understand the relation stuff, so maybe the question should have gone inside the `name` field of `hr.employee.relative`. Would it make sense to link there to a partner instead?

Reply:
I still think that creating a partner in order to enter a name is an over-complication. And how to limit access to that partner only to HR users?

Reply:
You can do both:

- To have an string for direct relations
- To have a partner_id link, with an onchange that dumps the name to `name` field, for advanced uses. If you put contact type as "private", it will only be seen by HR managers or the affected person (through portal).

Reply:
Done, please check if I got your idea correctly